.github/workflows/build-timestamped-master.yml

```diff
@@ -12,43 +12,7 @@ on:
     workflow_dispatch:
 
 jobs:
-    build:
-        runs-on: ubuntu-latest
-        if: github.repository_owner == 'ballerina-platform'
-        steps:
-            -   name: Checkout Repository
-                uses: actions/checkout@v3
-
-            -   name: Set up JDK 17
-                uses: actions/setup-java@v3
-                with:
-                    distribution: 'temurin'
-                    java-version: 17.0.7
-
-            -   name: Change to Timestamped Version
-                run: |
-                    startTime=$(TZ="Asia/Kolkata" date +'%Y%m%d-%H%M00')
-                    latestCommit=$(git log -n 1 --pretty=format:"%h")
-                    VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties | rev | cut --complement -d- -f1 | rev)
-                    updatedVersion=$VERSION-$startTime-$latestCommit
-                    echo $updatedVersion
-                    sed -i "s/version=\(.*\)/version=$updatedVersion/g" gradle.properties
-
-            -   name: Grant execute permission for gradlew
-                run: chmod +x gradlew
-
-            -   name: Build with Gradle
-                env:
-                    packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-                    packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-                    publishUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-                    publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-                run: ./gradlew publish --no-daemon --scan
-
-            -   name: Generate CodeCov Report
-                uses: codecov/codecov-action@v3
-            -   name: Upload Artifact
-                uses: actions/upload-artifact@v2
-                with:
-                    name: ballerina-runtime
-                    path: target/ballerina-runtime/
+    call_stdlib_workflow:
+        name: Run StdLib Workflow
```

Comment:
Shall we rename this to a more expressive name?

Reply:
+1, will update to following names:

Run Release Workflow
Run Central publish Workflow
Run PR build Workflow
Run Build Workflow